middleware/auth.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+const userService = require('../services/userService');
+
+const checkLogInToken = async (req, res, next) => {
+  try {
+    const accessToken = req.headers.authorization;
+    const secretKey = process.env.SECRET_KEY;
+
+    if (!accessToken) {
+      const error = new Error('NEED_ACCESS_TOKEN');
+      error.statusCode = 401;
+
+      return res.status(error.statusCode).json({ message: error.message });
+    }
+
+    const decoded = jwt.verify(accessToken, secretKey);
+
+    const user = await userService.getUserById(decoded.id);
+
+    if (!user) {
+      const error = new Error('USER_DOES_NOT_EXIST');
+      error.statusCode = 404;
+
+      return res.status(error.statusCode).json({ message: error.message });
+    }
+    req.userId = await decoded.id;
```

Comment:
user 확인 로직을 추가 하셨네요? 그렇다면 req.userId가 아닌 req.user로 바껴야 하게죠? 수정해 주세요.
```suggestion
    req.user = user;
```